newfile.py

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import logging
+
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
```

Comment:
On second thought, maybe this should be left out as there is no use of logging below